quinn-udp/src/unix.rs

```diff
@@ -293,7 +307,7 @@ fn send(
     }
 }
 
-#[cfg(any(target_os = "macos", target_os = "ios"))]
+#[cfg(any(target_os = "macos", target_os = "ios", target_os = "openbsd"))]
```

Comment:
`IP_RECVDSTADDR` seems to be defined in OpenBSD; should we add it to the cases where the `encode_ip` argument to `prepare_msg` is true below?

Reply:
good catch, yes.  testing that shows it also works.